onlinejudge_verify/verify.py

```diff
@@ -100,3 +104,10 @@ def main(paths: List[pathlib.Path], *, marker: utils.VerificationMarker, timeout
         # to prevent taking too long; we may fail to use the results of verification due to expired tokens
         if timeout is not None and time.time() - start > timeout:
             break
+
+    # failするテストがあったらexitcodeを1にする
+    if len(failed_test_paths) > 0:
+        logger.error('%d test failed', len(failed_test_paths))
+        for path in failed_test_paths:
+            logger.error('failed: %s', str(path))
+        sys.exit(1)
```

Comment:
```suggestion
        raise Exception('{} test failed'.format(len(failed_test_paths)))
```

よさそうですが、実はこれだとテストが失敗したときドキュメントがまったく生成されなくなります。以下のように外側で例外を待ち受けてるので、適当に何か投げましょう。
(「exitcode を 1 に」って言うと `sys.exit(1)` なのはそれはそうで、ごめんね)

https://github.com/kmyk/online-judge-verify-helper/blob/60247f9122849a2ad5bd385b94d356105e571f3a/onlinejudge_verify/main.py#L168-L172